corehq/form_processor/models.py

```diff
@@ -1203,15 +1203,16 @@ class LedgerValue(DisabledDbMixin, models.Model, TrackRelatedChanges, AbstractLe
     """
     objects = RestrictedManager()
 
-    # domain not included and assumed to be accessed through the foreign key to the case table. legit?
+    domain = models.CharField(max_length=255, null=False, default=None)
```

Comment:
It's the equivalent of this: https://github.com/dimagi/commcare-hq/pull/11545/files#diff-b4d5f5b9852b95f22afabd2a4da31057R394

Though I have been thinking about refactoring the whole thing to not be methods on the class but a helper in which case you could just pass in the domain separately